server/services/userService.test.ts

```diff
@@ -144,4 +148,22 @@ describe('User service', () => {
       expect(userClient.delete).toHaveBeenCalledWith(user.id)
     })
   })
+
+  describe('reference data', () => {
+    it('should return the lost bed reasons data needed', async () => {
+      const lostBedReasons = referenceDataFactory.buildList(2)
```

Comment:
I think it would be clearer if call to get probation regions returned probation regions as opposed to lostBedReasons